tests/console/validate_encrypt.pm

```diff
@@ -19,30 +19,72 @@ use strict;
 use warnings;
 use base "opensusebasetest";
 use testapi;
+use Test::Assert ':all';
+use scheduler 'get_test_data';
 
-sub run {
+sub parse_crypttab {
+    my $path = shift;
```

Comment:
I believe it's alright to have path to srypttab hardcoded, it's unlikely to change, but it's up to you.

Reply:
not hardcodeing it for the sake of not hardcoding anything does not make sense, it is like the name of the file or the comment, I agree.